gradle.properties

```diff
@@ -1,6 +1,6 @@
 GROUP_ID=com.xing.api
-VERSION=3.1.0
-VERSION_CODE=17
+VERSION=3.1.1
```

Comment:
Also not sure about when we should increase the version number, @MyDogTom can we do it in this PR and create a release when it is closed?

Reply:
Since we don't have any other changes planed, let's increase version right here.